src/jobs/sonarqube-audit.yml

```diff
@@ -0,0 +1,24 @@
+description: >
+  Job runs SonarQube scanner and upload results
+
+executor: node
+
+parameters:
+  sonarqube_settings_path:
+    type: string
+    description: "Path to file with sonarqube settings"
+
+steps:
+  - checkout
+  - attach_workspace:
+      at: .
+  - run:
+      name: Install Sonarqube scanner
+      command: |
+        wget https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.5.0.2216-linux.zip
```

Comment:
Can we keep sonarqube version as parameter? e.g.:
```
wget https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-<< parameters.version >>.zip
```
and  apply that to all cmds with version on it

Reply:
Adjusted